scripts/pr2/mannequine.py

```diff
@@ -73,6 +73,7 @@ def __init__(
         self.enable_mirror = mirror
         self.use_home_position = use_home_position
 
+        self.robot.angle_vector(ri.angle_vector())
```

Comment:
Not `ri` but `self.ri` is correct right ?  both `mypi` and `flake` is smart enough to detect this error. (see the CI result)

Reply:
You are right, `self.ri` is correct. I didn't check it enough and it got caught in the test. I realized again the importance of the test.